pkg/resources/convertible_test.go

```diff
@@ -51,6 +52,7 @@ func TestConvertToTerraformModel(t *testing.T) {
 			m, ok := resList[res.TypeName()]
 			if !ok {
 				t.Errorf("resource %s not found", res.TypeName())
+				return
```

Comment:
suggestion: this is redundant to t.Fatalf() which will log and exit. I fixed the panic here: https://github.com/mezmo/terraform-provider-mezmo/pull/148/commits/21d19c4a4c8b16637d6b655b78eafa7a734dc002

You can remove the return on line 46.